Scoop/Controller/Onboarding/AboutYouViewController.swift

```diff
@@ -55,14 +61,18 @@ class AboutYouViewController: OnboardingViewController {
     }
     
     private func setupStackView() {
-        let stackviewMultiplier = 0.10
-        let leadingTrailingInset = 20.0
+        var stackviewMultiplier = 0.20
+        let leadingTrailingInset = 32
         let spacing = 12.0
         let screenSize = UIScreen.main.bounds
-        let labelFont = UIFont(name: "Rambla-Regular", size: 16)
         let textFieldBorderWidth = 1.0
-        let textFieldCornerRadius = 8.0
+        let textFieldCornerRadius = 4.0
         let textFieldFont = UIFont(name: "SFPro", size: 16)
+        let textFieldHeight = 56
+        
+        if screenSize.height < 2000 {
```

Comment:
This feels weird? Are you sure we are going about laying out our UI correctly? What's the justification for this?